### ISSUE TITLE
[Rendering] Bug Title Here

### ISSUE THREAD

**Issue by UncleZen** [microsoft/AdaptiveCards#5945]
# Target Platform
- iOS
- Android

# Target Application
React Native app

# Problem Description
When starting the RN mobile app in a iOS simulator the error below is thrown:

Error: Unable to resolve module `AccessibilityInfo` from `node_modules/adaptivecards-reactnative/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.

See the screenshot below:

# Screenshot

<img width="430" alt="Screen Shot 2021-06-01 at 1 33 49 PM" src="https://user-images.githubusercontent.com/485880/120367292-b0393700-c2de-11eb-97f7-5bb8bda25169.png">

# Card JSON

Please provide the JSON of the Card that is being rendered

# Sample Code

Please provide any relevant code samples to help reproduce the issue

# Version of SDK

**React**: v. 16.13.1
**React Native**: v. 0.63.3
**adaptivecards-reactnative**: v. 2.2.8
**npm**: v. 7.7.6

**Comment by UncleZen** [comment]
Please note the we are getting similar error message when building the app on MS App Center. See below:

==============================================================================
Task         : Command line
Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
Version      : 2.182.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
==============================================================================
Generating script.
========================== Starting Command Output ===========================
[command]/bin/bash --noprofile --norc /Users/runner/work/_temp/4208411b-cf16-4cac-aced-ad4cd05a3b7d.sh
Found index.js for ReactNative index.
warning: the transform cache was reset.
                 Welcome to React Native!
                Learn once, write anywhere


error Unable to resolve module `AccessibilityInfo` from `node_modules/adaptivecards-reactnative/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*. Run CLI with --verbose flag for more details.
Error: Unable to resolve module `AccessibilityInfo` from `node_modules/adaptivecards-reactnative/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
    at ModuleResolver.resolveDependency (/Users/runner/work/1/s/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:186:15)
    at ResolutionRequest.resolveDependency (/Users/runner/work/1/s/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/Users/runner/work/1/s/node_modules/metro/src/node-haste/DependencyGraph.js:287:16)
    at Object.resolve (/Users/runner/work/1/s/node_modules/metro/src/lib/transformHelpers.js:267:42)
    at /Users/runner/work/1/s/node_modules/metro/src/DeltaBundler/traverseDependencies.js:434:31
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/runner/work/1/s/node_modules/metro/src/DeltaBundler/traverseDependencies.js:431:18)
    at /Users/runner/work/1/s/node_modules/metro/src/DeltaBundler/traverseDependencies.js:275:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/runner/work/1/s/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
##[error]Bash exited with code '1'.

**Comment by RebeccaAnne** [comment]
@IbrahimSulai Could you take a look at this one? Thanks!

**Comment by UncleZen** [comment]
@RebeccaAnne, @IbrahimSulai 

Please note that as an interim solution, I am deleting the node_modules folder within the adaptivecards-reactnative package folder post install:

"postinstall": "rm -rf ./node_modules/adaptivecards-reactnative/node_modules"

**Comment by jonmill** [comment]
Hi @UncleZen - the ReactNative renderer is an open-source project. In order to get help, please open the bug in the repo for that project specifically. Thanks!

https://github.com/bigthinkcode/adaptivecards
